### PR TITLE
Move PDF Generator to inside licence app

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: licences/preprod/db.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+resource "aws_security_group" "db" {
+  name        = "${var.app-name}-db-sg"
+  vpc_id      = "${aws_vpc.vpc.id}"
+  description = "${var.app-name} DB security group"
+
+  ingress {
+    from_port       = 5432
+    to_port         = 5432
+    protocol        = "tcp"
+    cidr_blocks     = ["${var.ips["office"]}/32", "${var.ips["mojvpn"]}/32"]
```
Comment (mattops) [inline licences/preprod/db.tf]: Do we really want to open up access to preprod db from studio/vpn?  I'd guess this was ok in dev/stage.

File: licences/preprod/main.tf [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+    namespace = "aws:elasticbeanstalk:application:environment"
+    name      = "ADMIN_API_CLIENT_SECRET"
+    value     = "${data.aws_ssm_parameter.admin-api-client-secret.value}"
+  }
+
+  tags = "${var.tags}"
+}
+
+locals {
+  cname = "${replace(var.app-name,"-prod","")}"
```
Comment (mattops) [inline licences/preprod/main.tf]: what's this for?

File: licences/preprod/main.tf [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+  }
+  setting {
+    namespace = "aws:elasticbeanstalk:application:environment"
+    name      = "SESSION_SECRET"
+    value     = "${random_id.session-secret.b64}"
+  }
+  setting {
+    namespace = "aws:elasticbeanstalk:application:environment"
+    name      = "NODE_ENV"
+    value     = "production"
```
Comment (mattops) [inline licences/preprod/main.tf]: should this be `production`?  Assume this is also incorrect in dev/stage environments

File: licences/preprod/db.tf (hunk 1 above)
Comment (Mjwillis) [reply licences/preprod/db.tf]: yes, I think so as we have no bastion box to jump from - so if a live production issue occurs we cannot get DB access.  Furthermore we need to run the migration script against the db from a laptop

File: licences/preprod/main.tf (hunk 2 above)
Comment (Mjwillis) [reply licences/preprod/main.tf]: removes the -prod from the cname

File: licences/preprod/main.tf (hunk 3 above)
Comment (Mjwillis) [reply licences/preprod/main.tf]: No production is correct for all environments, except local running

**Pull request merged**